CreeDictionary/CreeDictionary/templates/CreeDictionary/base.html

```diff
@@ -68,13 +68,13 @@
     {% endblock %}
   </div>
 
-  <section class="search-results">
+  <div class="search-results">
```

Comment:
This isn't really a document section, and validators complain if a `<section>` does not have a heading tag (e.g., `<h2>`, `<h3>`) within it.